axum/src/docs/routing/route.md

```diff
@@ -60,6 +60,16 @@ Wildcard captures can also be extracted using [`Path`](crate::extract::Path).
 Note that the leading slash is not included, i.e. for the route `/foo/*rest` and
 the path `/foo/bar/baz` the value of `rest` will be `bar/baz`.
 
+Example:
+
+```rust
+let app = Router::new().route("/*key", get(handler));
+
+async fn handler(path: axum::extract::Path<String>) -> String {
+    path.0
+}
+```
+
```

Comment:
Lets combine it with the paragraph above

```suggestion
Wildcard captures can also be extracted using [`Path`](crate::extract::Path):

```rust
use axum::{
    Router,
    routing::get,
    extract::Path,
};

let app = Router::new().route("/*key", get(handler));

async fn handler(Path(path): Path<String>) -> String {
    path
}
```

Note that the leading slash is not included, i.e. for the route `/foo/*rest` and
the path `/foo/bar/baz` the value of `rest` will be `bar/baz`.
```